components/channel_notifications_modal/components/describe.jsx

```diff
@@ -7,7 +7,7 @@ import {FormattedMessage} from 'react-intl';
 
 import {IgnoreChannelMentions, NotificationLevels, NotificationSections} from 'utils/constants.jsx';
```

Comment:
Rather than adding values to `constants.jsx` I've converted this prop to a boolean.